src/overlays/actors/ovl_Obj_Hsblock/z_obj_hsblock.c

```diff
@@ -43,11 +44,11 @@ static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneDownward, 2000, ICHAIN_STOP),
 };
 
-CollisionHeader* D_80B940DC[] = { 0x06000730, 0x06000730, 0x06000578 };
+CollisionHeader* D_80B940DC[] = { &gObjDHsblockCol1, &gObjDHsblockCol1, &gObjDHsblockCol0 };
```

Comment:
```suggestion
static CollisionHeader* sCollisionHeaders[] = { &gObjDHsblockCol1, &gObjDHsblockCol1, &gObjDHsblockCol0 };
```
or appropriate name.